partner_create_by_vat/models/res_partner.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+
+import requests
+
+from string import maketrans
+from stdnum.eu.vat import check_vies
+from lxml import html
+
+from openerp import models, fields, api, _
+from openerp.exceptions import except_orm
+
+CEDILLATRANS = maketrans(u'\u015f\u0163\u015e\u0162'.encode(
+    'utf8'), u'\u0219\u021b\u0218\u021a'.encode('utf8'))
+
+
+def getMfinante(cod):
+    headers = {
+        "User-Agent": "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.0)",
+        "Content-Type": "multipart/form-data;"
+    }
+    params = {'cod': cod}
+    res = requests.get(
+        'http://www.mfinante.ro/infocodfiscal.html',
+        params=params,
+        headers=headers
+    )
+    res.raise_for_status()
+
+    htm = html.fromstring(res.text)
+    # sunt 2 tabele primul e important
+    table = htm.xpath("//div[@id='main']//center/table")[0]
+    result = dict()
+    for tr in table.iterchildren():
+        key = ' '.join([x.strip() for x in tr.getchildren()[
+                       0].text_content().split('\n') if x.strip() != ''])
+        val = ' '.join([x.strip() for x in tr.getchildren()[
+                       1].text_content().split('\n') if x.strip() != ''])
+        result[key] = val.encode('utf8').translate(CEDILLATRANS).decode('utf8')
+    return result
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    name = fields.Char('Name', required=True, select=True, default=' ')
+
+    @api.one
+    def button_get_partner_data(self):
+        def _check_vat_ro(vat):
```

Comment:
Yes, but I was just talking about the vat argument in the signature of _check_vat_ro(vat). It is unused in the rest of this inline function so I was wondering if you meant to refer to something else or if it can be left out entirely.